scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/PPMLContext.scala

```diff
@@ -36,21 +36,50 @@ import org.apache.hadoop.fs.Path
  * @param kms
  * @param sparkSession
  */
-class PPMLContext protected(kms: KeyManagementService, sparkSession: SparkSession) {
+class PPMLContext protected(kms: KeyManagementService = null, sparkSession: SparkSession = null){
 
   protected var dataKeyPlainText: String = ""
+  protected val multiKms = new scala.collection.mutable.HashMap[String, KeyManagementService]
```

Comment:
As we have both kms and multikms variables. We need to ensure we don't misuse them.

Reply:
You can use this multiKms to take the funtion of `kms`, so you can delete the old `kms`.